src/components/Text/Text.tsx

```diff
@@ -8,7 +8,7 @@ import { ComponentVariablesInput, ComponentPartStyle } from '../../../types/them
 
 export interface ITextProps {
   as?: any
-  atMention?: boolean
+  atMention?: 'me' | 'other'
```

Comment:
I would think that we would still support the `boolean` case here.  Some themes (like GitHub) don't have the concept of `me` vs `other`.  In those cases, themes have the need of styling based on the presence of this prop alone (boolean).

```jsx
<Text atMention>@someone</Text>
<Text atMention="me">@me</Text>
<Text atMention="other">@other</Text>
```

What these `atMention` values look like or whether or not they look different between their usages is up to the theme.

Reply:
Actually, thinking through https://github.com/stardust-ui/react/pull/277#discussion_r220693137 more carefully I think we only need `boolean` and `me`, correct?  An `atMention` is a generic mention while `atMention="me"` is the only specifier needed for custom formatting.